services/user.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
Using `User` directly is not recommended. Instead, use `get_user_model()` to ensure compatibility with custom user models. This aligns with the checklist item: 'Use get_user_model() instead of User, it is the best practice.'